### ISSUE TITLE
Add T9432

### ISSUE THREAD

**Issue by Nsl106** [google/fonts#6909]
**Font Project Git Repo URL:** https://github.com/Nsl106/T9432

T9432 is a pixelated typecafe based on [Silkscreen](https://github.com/googlefonts/silkscreen/) designed for use by FRC Team 8-Bit

**Requirements:**

Google Fonts will publish only fonts that match its requirements. Please familiarize yourself with the complete documentation in the [Google Fonts Guide](https://googlefonts.github.io/gf-guide/) (GF-Guide) and ensure your font project complies with them before submitting the font family. You can also use the [Google Fonts Project Template](https://github.com/googlefonts/googlefonts-project-template), which will help you create a repository that follows the needed structure and includes build requirements.

By filling this issue, you can confirm the project meets the  requirements (by ticking the cases or putting x between the square brackets in text mode):

- [x] I read, agree, and comply with the full [contributing requirements](https://googlefonts.github.io/gf-guide/index#pre-production-getting-your-fonts-ready-for-gf)
- [x] The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
- [x] There is no "Reserved Font Name" mention in the OFL license
- [x] Family name is unique according to [namecheck.fontdata.com](https://namecheck.fontdata.com/)
- [ ] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/)
- [x] The repository respect the [upstream repo structure](https://googlefonts.github.io/gf-guide/upstream.html)
- [x] The source files are available in the repository
- [x] I am the author of the font, or the author agreed to this submission
- [x] I will maintain the repository and participate in the onboarding process (addressing, solving, and responding to issues, merging pull requests, etc)

**Image:**
![T9432](https://github.com/google/fonts/assets/87742096/4c3cbdaf-6669-4e00-8d58-899b0a15f82f)

**Comment by Nsl106** [comment]
This font doesn't support the GF Latin Core glyphset because of it's pixelated nature. I can't make the glyphs with diacritics and such fit in the same dimensions/style as the other characters. If exceptions to this can't be made, no worries. This is my first time submitting a font here so I'm still learning how it all works!

**Comment by eliheuer** [comment]
@Nsl106 Thank you for the submission!
How is this typeface different from [Silkscreen](https://github.com/googlefonts/silkscreen/)?
Is the main difference that it adds lowercase drawings?

I used to make pixel fonts when I worked in the game industry, and I don't understand why it is not possible to support the full GF Latin Core glyphset. Even if the font is low resolution, you can still do a drawing for each code point, it is just sometimes very abstract due to the low resolution.

**Comment by Nsl106** [comment]
Compared to Silkscreen, most of the characters are thinner/sharper which was the main thing I wanted from this project. I'd be happy to try and make the other characters, my concern is that they may end up being too abstract. The other reason I didn't add them is because I would have to shrink the other characters 40% by adding two empty rows of pixels that wouldn't be used all that often.

Here are more of the other characters if you're interested:
![image](https://github.com/google/fonts/assets/87742096/c495cb88-8a29-4529-8da5-89f0d45b028d)

**Comment by davelab6** [comment]
How does this compare to the kernel set? And silkscreen itself?

**Comment by davelab6** [comment]
Overall I'm supportive, we have similar pairs of caps and bicameral fonts in barrio and a few others I think

**Comment by Nsl106** [comment]
It should have all the characters in the kernel set, but I'm not 100% sure. Is there a tool to check that in an automated way? I think silkscreen has more of the glyphs with diacritics because it's characters are a higher "resolution", but I'm also not sure what the best way to compare them is.

**Comment by eliheuer** [comment]
@Nsl106 Yes, there is a tool to check that in an automated way. If you are using Glyphs, put [this file](https://github.com/googlefonts/googlefonts-project-template/blob/main/sources/CustomFilter_GF_Latin_All.plist) in the same directory you are working from, and you will get a fillter list on the sidebar telling you what you are missing:

<img width="287" alt="Screenshot 2023-12-07 at 7 25 09 AM" src="https://github.com/google/fonts/assets/5162664/1a15c2a2-ace8-4227-af8d-3cd9965153c2">

**Comment by eliheuer** [comment]
Also see the glyphset section of the Google Fonts guide if you need more info: https://googlefonts.github.io/gf-guide/

**Comment by eliheuer** [comment]
@Nsl106  We reviewed this and it has been added to the queue for potentially onboarding early (Q1 2024) next year.

If you have not yet looked at the specifications and requirements in the Google Fonts documentation, you can find that here: https://googlefonts.github.io/gf-guide/

Me or someone else can help you with onboarding soon. If you have any questions, please ask them here.

I can check the glyph-set and make sure nothing is missing, so no need to worry about that yet. I'll make a post here if something is missing.

**Comment by Nsl106** [comment]
Awesome, thanks!
